### ISSUE TITLE
4.0 32 hardened  (for use in making a cloud pentoo) from i686 hardened. 

### ISSUE THREAD

**Issue by necrose99** [pentoo/pentoo-livecd#4]
not quite all that healthy on install 
just a mention, 
manual unpack of sqaushfile to root not so good either | > pentoo-i686-hardened.tar.gz 
grr nuts.... 

BUILD: https://hub.docker.com/r/necrose99/gentoo-docker-x86/  Add Gentoo portage 
Pentoo-686 (do Binary installs && Swap profile to build emerge world wala dockers Cloud farm will do the dirty work for me. daily::smiley:: ) **more to the point a fresh Chroot from the stock pentoo/daily's**  
{ undocker to chroot and slap some iso's [Mudlers Sabayon un-Docker info , ](https://hub.docker.com/r/sabayon/spinbase-amd64/) --Sure to give brain freeze-- 
docker is like git/diff too , i can fork your container make a few mods and send you back the container I forked (which is only DIFFS less un-docker'ed to a chroot --yes brainfreeze--- 
**for testing new ebuilds or bump testing , i can have root all day if i fork your container 2 x's for my own spare , if i break it , it wont really damage much, its on a branch like git. **

Also from LXC/Docker Remote x11 is possible . ( bump testing of major x11 components) 
so I can add crap to a container , ie more ebuilds and shove the throttle to full and dive off the train before it slams into the wall , **rinse and repeat for Q/A. ...** 
but any bump tested fails I can just ship back test logs etc ie on a branch and not the Entire VM. 
 

however having a source tarball  stage 4/5 CHROOT would save much effort for this one.  
(https://hub.docker.com/r/necrose99/pentoo-i686-hardened-docker/) but i still can rig it from Gentoo if I must, 

_**can layer docker builds like a cake**_ ::grinning::
gentoo/stage3-amd64-hardened or 
gentoo/stage3-amd64
& gentoo/portage
https://hub.docker.com/r/necrose99/pentoo-i686-hardened-docker/
or pentoo/amd64-hardened 

**A:  add to build gentoo/stage3-amd64-hardened or 
gentoo/stage3-amd64
B: add to build gentoo/portage
C: add layman Pentoo overlay and emerge packages  
D: emerge rebuild world if wanted, hub.docker.com will do this for you daily. 
E: done.** 
(Pull and un'docker /Spank to iso daily if wanted) 

Abuse the version-able Chroot to death in testing... package bumps or even Witchcraft testing . 
but a reliable Pentoo-{$ARCH}{$VARI) conatiner for testing probing or 
($ARCHAMD else i686)  ($VARI vanilla or hardened) 

docker is also capable of being clustered, ie one mean server and a buch of dell Optiplex or surplus pc's acticing like distcc but for everything, cracking keys anyone, compiles etc. ? ,emerge/compile in parallel

**Comment by necrose99** [comment]
triv item for skell  can cad add a start network mgr 
lazy tweak but friendlier. 

[Desktop Entry]
Version=1.0
Type=Application
Name=Start-Networkmanger
Comment=Start up Network Manager
Exec=/etc/init.d/NetworkManager start
Icon=network-transmit-receive
Path=
Terminal=true
StartupNotify=true
GenericName=Start up Network Manager

**Comment by ZeroChaos-** [comment]
please, one bug per issue, not a random collection

as for the second comment, have you tested this desktop file?  I like the idea.

**Comment by necrose99** [comment]
**yes  i created it for laziness** . while banging out tarball for pentoo-docker testing. just haven't made one for Stopping network stack. a few reboots later ..shiznit openrc how to start network mgr i just stated x ....  ( had you a pentoo-skel repo I'da added it )

 may just hate **virtualbox 5.08 for windows as host.** (building vbox 5.08 support's fails (upstream issue)  also , cant see if that fixes issue nuts)  --**I really need to recover/replace 2nd laptop HDD**--
rc4 x86 hardened  i can screen share , but installer freezes or crashes and or is picky on partitions in **Very specific order...  **
/ ,swap ,   bookable system .... snap a tarball > inject to docker hub , nuts... 

**had I not been trying to harvest A working pentoo Install to tarball** for docker file , and pentoo cloud build  I would have not likely been needing to test iso. 
(upstreem bug gentoo/docker seemes to quirk out for i686-hardened autobuild thus forcing me to get a squash-file , instead of just building from pentoo-OL on a daily basis ) (dockerhub doing them for me) and anyone can then pull.  had their been Stage-5 tarballs It'd be even easier to push to container. 


**however***  for Pure vanilla x86  dockerhub/necrose99 
, AMD64 and AMD64 hardened I can just Call the Gentoo containers on docker hub 
add in layman and provisioning scripts , and use current package lists & or version >= 
can rig makefile and or kill rysnc licencing so container will build all the nice crap.

**Bonus Prize is fork-able**  (like github branches) can do regular use updates or multiple testing and or insane-O mode testing using same chroot AUFS/OverlayFS branches 
Can be installed on say a debian/redhat/etc box with x11 etc as a container , a few quirks , but for testing newer xfce etc against genmenu , also a safe alternative. (few other things to do till that) 
Gives Devs/Users ability to collaborate in unique ways not possible before when testing new Goodies. for posible inclusion in repo. 
**Use Pentoo as basis for automated Gentile-Ingesting of Own Infrastructure  Pen-SASS , or add in IDS modules for a one stop Solution etc. 
*** Daily or weekly DEV-Testing-ISO's 

**Devious prize** is you can install Pentoo in a target of a pentest environment  if docker/server (cloud security Pentesting a sore point in some companies more like a Redwood with a Highway hole cut into it and drive a semi through)  **Tall Wooden Horses are more obvious***  In theory oh so possible.

**Comment by ZeroChaos-** [comment]
I pushed the desktop file, thanks.

**Comment by necrose99** [comment]
installer is now working   unfortunately my docker experiments have failed. 
using the gentoo from scratch Docker i found of someone esle, were as 4-6 split tarballs. 
but maintenance should be simpler , however , will pull into a builder to rebuild all , so thus its fresh. 
mainly the echo message and it dies , before stripping latest iso, to squashfs > / granted i can likely move to a provishion.sh , 
CMD (/faacepalm)  RUN echo "msg"  basicly logs the unatteneded to screen log , on git hub-commit , when the builder decided's to auto-build @ dockerhub. 
IF still keep getting derpy results i'm going to switch to other template. 
Docker-hub already has a hard-on for Gentoo-x86-harded namely epic grand fails. 

but if it fails once more i'm going for tarballs and the from scratch gentoo template , with Busybox bzcat to dump 5 tarballs to </root-pentoo-fs/>  @ / 

drone CI /Cricle CI  can use a pentoo  Docker conainer , i've been making to , build on commit to github. best of all Since a few of the ebuild's I'm Frankenstein'ing , are for Pentoo, can see if they work , even if doing a browser based fix from any o/s 

then once builds are stable , one can get the container to self destruct but keep all the packages, as an option. or my main case CHROOT 

but if you have a crap ton of Cheep PC's https://www.youtube.com/watch?v=9SpQ1fRQAto 
XEN baseed cluster , but with **docker... cluster.**..   DISTCC??? but why not **EVERYTHING**
Keys I cant break GPU/Cheep-pcs cluster on docker with 1 big beefy server....   <!@----Distributed----@> wep/wpa keys and or rainbow tables, without bogging down laptop. 

Once builds are stable UN-fork repo's into combined , **ADD WIKI**
my Main use case is a test chroot , as to not kill main environment, 
however having the ability to do cluster computing is a nicety. 
PUSH the repo (chown) to Pentoo
